.appveyor.yml

```diff
@@ -5,15 +5,13 @@ environment:
     - GENERATOR : "Visual Studio 14 2015 Win64"
       QTDIR: C:\Qt\5.6.3\msvc2015_64
       PLATFORM: x64
-      BUILD_TESTING: ON
     - GENERATOR : "Visual Studio 14 2015"
       QTDIR: C:\Qt\5.6.3\msvc2015
       PLATFORM: Win32
-      BUILD_TESTING: ON
     - GENERATOR : "MinGW Makefiles"
       QTDIR: C:\Qt\5.6\mingw49_32
       PLATFORM: x86
-      BUILD_TESTING: OFF
+      CMAKE_CXX_FLAGS_INIT: -DCATCH_CONFIG_NO_CPP11_TO_STRING
```

Comment:
Is this thing documented anywhere?

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS_INIT.html